roles/pulp_devel/tasks/install_docs_requirements.yml

```
@@ -5,20 +5,23 @@
     state: present
     virtualenv: '{{ pulp_install_dir }}'
     virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
+  when: pulp_requirements_dir is defined
```

Are you sure these 3 statements will work? defaults/main.yml makes me think it would be an empty string.

Heres's syntax that will check for being defined and not empty or Null.
```suggestion
  when: pulp_requirements_dir | default('', true) | length > 0
```